MovieHubRx/Data/APIService/Urls.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Urls.swift
+//  MovieHubRx
+//
+//  Created by Duy Nguyễn on 14/05/2024.
+//
+
+import Foundation
+
+struct Urls {
```

Comment:
chỗ này k nhất thiết phải dùng struct, enum cũng được rồi e
```suggestion
struct APIUrls {
```